popde/density_estimate.py

```diff
@@ -1,3 +1,128 @@
+class SimpleGaussianKernelDensityEstimation:
+    """
+    Kernel Density Estimation (KDE) using Gaussian Kernels
+
+    Parameters:
```

Comment:
If the parameters are given in the init method, they should not be described here as well. There is no point having the same comments twice

Reply:
Yes, this was repetitive and removed now.